fastseq_cli/transformers_generate.py

```diff
@@ -318,7 +322,8 @@ def run_generate():
                         default=-1,
                         required=False,
                         help="min length for decode")
-
+    parser.add_argument("--causal_lm", action="store_true")
+    parser.add_argument("--output_summaries_only", action="store_true")
```

Comment:
why need this option?

Reply:
GPT2 uses AutoModelForCausalLM instead of AutoModelFor Seq2SeqLM in order to load the pretrained model. Previously this was conditioned on model_name, however this assumed that only huggingface's default gpt2 model will be used and didn't support using a path to the pretrained model